### DISCUSSION TITLE
Curso Fullstack .NET - Identity no Blazor

### DISCUSSION THREAD

**Discussion by thomasmoreira** [balta-io/forum#590] [Tira Dúvidas] [answered]
Sobre a aplicação web, no curso foi usado o modelo wasm. Se eu usar o modelo auto, como ficaria a autenticação com cookies? Seria a mesma implementação? Caso não seja, teria algum exemplo?

Obrigado!

**Comment by andrebaltieri** [comment, marked as answer]
Bom dia, @thomasmoreira, como vai?

Fica sim, tanto Server quanto Auto seriam implementações bem diferentes.

Sobre o exemplo, o próprio Blazor já traz:
```
dotnet new blazor -o SeuApp --auth Individual
```

O comando acima vai trazer um exemplo com autenticação!
